packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -32,6 +32,19 @@ function TestWrapper() {
 	);
 }
 
+/**
+ * Returns the first found popover element up the DOM tree.
+ *
+ * @param {HTMLElement} element Element to start with.
+ * @return {HTMLElement|null} Popover element, or `null` if not found.`
+ */
+function getWrappingPopoverElement( element ) {
+	if ( element.classList.contains( 'components-popover' ) ) {
+		return element;
+	}
+	return getWrappingPopoverElement( element.parentElement );
+}
```

Comment:
This could be implemented as just `return element.closest( '.components-popover' )`. Your version will also crash when the popover element is not found, because it will reach the top of the DOM tree where `.parentElement` is `null` and then `element.classList` will fail.

Reply:
Sounds good. I've used `.closest()` in 1f7ee28d5606de43257516e6312ecd7b5edb4d8d. Wanted to point out that I kept the helper function here because it also helps satisfy the `no-node-access` ESLint rule.